sdk/storage/azure-storage-blob/tests/test_get_blob.py

```diff
@@ -142,11 +151,14 @@ def test_get_blob_to_bytes(self, storage_account_name, storage_account_key):
         content = blob.download_blob(max_concurrency=2).readall()
 
         # Assert
-        self.assertEqual(self.byte_data, content)
+        assert self.byte_data == content
 
     @pytest.mark.live_test_only
```

Comment:
Sorry, this may create a bit of work, but this test (and likely others) does not need to be Live Only. It has the comment `parallel tests introduce random order of requests, can only run live` but this test is not parallel since it does not use `max_concurrency` > 1 anywhere (like the test about it does). I saw this quite a bit in some other files too and I think it was just copy/pasted without consideration before.

I think this is something we should be addressing as part of this migration so would you mind recording this test and deleting that comment. Additionally, can you have a quick scan through other Live Only tests here to see if any other fall into this category?

Reply:
Good point. Will audit the Live Only tests and get rid of it accordingly 😄